lib/rubocop/cop/naming/rescued_exceptions_variable_name.rb

```diff
@@ -56,7 +56,11 @@ def on_resbody(node)
         private
 
         def preferred_name
-          @preferred_name ||= cop_config.fetch('PreferredName', 'e')
+          @preferred_name ||= begin
+            name = cop_config.fetch('PreferredName', 'e')
+            name = "_#{name}" if variable_name.to_s.start_with?('_')
```

Comment:
I'm not sure how I could use [this existing method][1] to avoid repeating code here 🤔 

[1]: https://github.com/rubocop-hq/rubocop/blob/master/lib/rubocop/cop/variable_force/variable.rb#L78-L80

Reply:
The small amount of repetition is fine here.